pkg/test/collector_intermediate_test.go

```diff
@@ -56,14 +56,14 @@ dataPacket 2:
 	"destinationTransportPort": 5678
 	"protocolIdentifier": 6
 	"flowEndSeconds": 1257895000
-	"packetTotalCount": 400
-	"packetDeltaCount": 200
+	"packetTotalCount": 1000
```

Comment:
why same? I know it can be similar but as they are not synchronized, they may not be exactly the same stats. 

Does the order issue with tcp still exists? I am just curious.

Reply:
I was just trying to simulate the condition. since they may not be exactly the same, I changed it to different values.
For order issue in tcp, it will not cause order problem in collector since data are received in order. Because aggregation has two workers, the sequence of processing will differ. I have changed the worker number to 1. It should be working now.

Reply:
Processing with two workers is an important point. We should allow this either sending all flow records of an individual flow to one worker or update the flow records carefully if flow records can be processed by multiple workers.
Should we update only if the value is more than current value for Total stats?

Reply:
Yes it makes sense to update total stats when value is larger. Will update the code then. Thanks!